.github/workflows/fossa-scanner.yaml

```diff
@@ -0,0 +1,14 @@
+name: "FOSSA Scanner"
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
```

Comment:
workflow_dispatch is added for testing purposes. This can be removed later after flow is confirmed.